experiments/run_bulk_evict_insert.py

```diff
@@ -15,12 +15,12 @@
 
 base_window_sizes = [4*u.MB]
 
-base_iterations = 10 * u.MILLION
+base_iterations = 5 * u.MILLION
```

Comment:
Is 5m going to work better than 10m? It seems that 10m was already too small and we were hoping for something bigger.

Reply:
These iteration counts are not final - I made them small on purpose so that I could get some data to test the scripts. We'll need to iterate on them in the future, so I don't consider their current values important.